src/main/resources/db/migration/V1_4__create_index_melding.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX IX_MELDING_CONVERSATION_REF ON MELDING (conversation_ref);
```

Comment:
Skal jeg legge til flere indexer? Ser vi har spørringer på `uuid` også, og `conversation_ref` + `personident` sammen. Jeg er ikke såå stødig på indexer, men trenger man en egen index på det når man kombinerer oppslag på to felter med `AND`?

Reply:
uuid trenger man ingen egen indeks på dersom den er deklarert som unique: for da lager Postgres automatisk en indeks i kulissene.

Reply:
Vi kan se litt hvordan det går med responstider på de aktuelle query'ene, men jeg tipper vi ikke trenger noen kombinert indeks fa conversation_ref + personident. Dersom det er indeks på disse kolonnene hver for seg vil Postgres selv velge hvilken av dem som er best å bruke.